### ISSUE TITLE
Network load vs Node load

### ISSUE THREAD

**Issue by darice11** [storj/storj#4043]
On the web page for my node, I can see how busy it is. Looking at the Bandwidth used each day, Egress and ingress plus the space used and the amount of trash, I can tell how busy my node is.  This information does not however tell me how busy my node is in comparison to the rest of the network.

Here is the thing, I have good information on how busy my node is, but no information on how busy the overall network is, and therefore no information on how busy my node should be relative to other nodes or relative to historical Storj network loads. So even if my node is working, I don’t know what workload it should be processing. I could have network congestion, a bottleneck in my platform or my ISP is throttling my uplink/downlink to a satellite because it’s using to much bandwidth (fair usage policy) and my node is not able to perform at max.

In summary I need to know what load my node should be processing and I suggest historical network loading data would enable me to map my nodes load in comparison. At a minimum I suggest a line on my Bandwidth used this month graph showing daily Storj network load, so if my bandwidth load line doesn’t trend in parallel with the Storj network load I can surmise something is affecting my platform’s performance and start looking.

**Comment by stefanbenten** [comment]
Hey @darice11 ,

we just announced https://stats.storjshare.io, which provides quite a few interesting metrics ( bandwidth used over last 30 days, etc), which in correlation with the node count can be used to calculate your requested values.
We do not intend to create graphs for every possible interest and let people gather the metrics themselves and make the charts they care about :+1: 

Let me know, if this doesnt resolve the issue, else i would propose to close it now.

**Comment by darice11** [comment]
Thanks Stefan. The raw data will take a bit of work for easy view but it's a good start.

Cheers,

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
